parsing/dialects/dialect1/parserImpls.ftl

```diff
@@ -1801,19 +1801,12 @@ SqlNode InlineCaseSpecific() :
     (
         value = StringLiteral()
     |
-        LOOKAHEAD( CompoundIdentifier() <LPAREN> [ <NOT> ] (<CASESPECIFIC>|<CS>) <RPAREN> )
+        LOOKAHEAD( CompoundIdentifier() CaseSpecific() )
         value = CompoundIdentifier()
     )
-    <LPAREN>
-    [ <NOT> { not = true; } ]
-    (
-        <CASESPECIFIC>
-    |
-        <CS>
-    )
-    <RPAREN>
+    not = CaseSpecific()
```

Comment:
Rather than having CaseSpecific() return a "not" (which is a little counterintuitive), what if it just returned the constructed "SqlCaseSpecific()" object?